recipes/xerces-c/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+if [ `uname` == Darwin ]; then
+	export CC=clang
+	export CXX=clang++
```

Comment:
I am fine with that. I just wonder if we should do that with the rest of the (hdf5) dependencies.